docker/pg_tests/scripts/tests/test_functions/backup_and_restore_test.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/sh
```

Comment:
This seems like the script from the remote backup patch :)

Reply:
Ouch. I removed it.